tests/test_ace.rb

```diff
@@ -112,15 +112,20 @@ def test_addrs
     assert_equal(val, a.src_addr)
     assert_equal(val, a.dst_addr)
 
+    config_no_warn('object-group ip address my_addrgroup_ipv4')
```

Comment:
Did we get confirmation that this object being present is intended/expected behavior in `I6`?